recipe/meta.yaml

```diff
@@ -10,36 +10,90 @@ source:
   fn: boltons-{{ version }}.tar.gz
   url: https://github.com/mahmoud/boltons/archive/{{ version }}.tar.gz
   sha256: {{ hash }}
+  patches:
+    # https://github.com/mahmoud/boltons/commit/270e974975984f662f998c8f6eb0ebebd964de82
+    # Can be removed in next version (>21.0.0)
+    - patches/00_fix_py310_tests.patch  # [py>39]
 
 build:
   noarch: python
```

Comment:
```suggestion
```

Reply:
I am open on this one though. A collection of scripts seem to fit a noarch package.
If you decide to keep noarch, I think you need to set python >=3.4 in host and run.

Reply:
I think for consistency, I will remove the `noarch` unless you feel strongly I should leave it.

Reply:
Works for me!